test/support/commands.js

```diff
@@ -4,22 +4,26 @@ Cypress.Commands.add('getPreviewPage', (pageNum) => {
     cy
         .get(`.page[data-page-number=${pageNum}]`)
         .as('previewPage')
-        .find('[data-testid="page-loading-indicator"]')
+        // Adding 10s timeout here because sometimes it takes more than the Cypress
```

Comment:
Looks like the duration in the comment is outdated. Probably easier to just remove it?